### ISSUE TITLE
Untitled

### ISSUE THREAD

**Issue by marazmiki** [SmileyChris/django-mailer-2#8]
it seems to me, there is a bug in [django_mailer/engine.py](https://github.com/SmileyChris/django-mailer-2/blob/master/django_mailer/engine.py#L21) at line 21:

```
LOCK_PATH = getattr(settings.LOCK_PATH,
                    os.path.join(tempfile.gettempdir(), 'send_mail'))
```

IMHO, this code must be rewritten as follow:

```
LOCK_PATH = getattr(settings, 'LOCK_PATH',
                    os.path.join(tempfile.gettempdir(), 'send_mail'))
```

**Comment by SmileyChris** [comment]
Thanks, fixed in 9ec04e0040be6d0cb62d31354a2c8ecd914e0990
(We can rely on settings.LOCK_PATH existing, it's the app settings not the global ones, but I shouldn't have been using getattr)

**Comment by marazmiki** [comment]
Now we haven't AttributeError exception, all works fine
Thanks
